### PR TITLE
Added remove license command

### PR DESCRIPTION
**Pull request by rjd22**
@arnovr @lucasvanlierop can you please review

### CODE REVIEW

**Comment by arnovr** [comment]
Je build faalt!

File: README.md [hunk 1]
```diff
@@ -44,6 +44,17 @@ $command->ssoHandshakeRequesterIp = '127.0.0.1/32'; // sso IP address/mask of ha
 $shopLicense = $dreamCommerceClient->createLicense($command);
 ```
 
+### Removing a License
+To remove a DreamCommerce license there is a `removeLicense` method. I will return a boolean if the removal has
```
Comment (arnovr) [inline README.md]: Will you return a boolean? How nice, i thought the method would do that :neckbeard:

File: src/LicenseManagerClient.php [hunk 2]
```diff
@@ -44,6 +45,19 @@ public function createLicense(CreateLicenseCommand $command)
...
+     * @param RemoveLicenseCommand $command
+     * @return boolean
+     */
+    public function removeLicense(RemoveLicenseCommand $command)
+    {
+        $licenseRemoved = $this->connection->sendRequest('removeLicense', [
+            'licenseId' => $command->shopLicenseId
+        ]);
+
+        return $licenseRemoved === 'true';
```
Comment (arnovr) [inline src/LicenseManagerClient.php]: I personally prefer this:

```php
        $licenseRemoved = (boolean) $this->connection->sendRequest('removeLicense', [
            'licenseId' => $command->shopLicenseId
        ]);
```

But we had this discussion already or not?

File: tests/LicenseManagerClientTest.php [hunk 3]
```diff
@@ -61,6 +62,29 @@ public function shouldReturnAccountInformationWhenCreatingAnAccount()
     /**
      * @test
      */
+    public function shouldReturnResultWhenRemovingAnAccount()
```
Comment (arnovr) [inline tests/LicenseManagerClientTest.php]: Fail flow is missing?

Comment (arnovr) [inline tests/LicenseManagerClientTest.php]: And maybe define what it returns
shouldReturnSuccessWhenRemovingAccount

File: README.md (hunk 1 above)
Comment (rjd22) [inline README.md]: Yes, when you execute the method I will come running to you and slap a `0` or an `1` sticker on your forehead as a response.

File: tests/LicenseManagerClientTest.php [hunk 4]
```diff
@@ -61,6 +62,29 @@ public function shouldReturnAccountInformationWhenCreatingAnAccount()
...
+            ->withArgs([
+                'removeLicense', [
+                    'licenseId' => 'shop-license-id'
+                ]
+            ])
+            ->andReturn('true')
+            ->once();
+
+        $command = new RemoveLicenseCommand();
+        $command->shopLicenseId = 'shop-license-id';
```
Comment (arnovr) [inline tests/LicenseManagerClientTest.php]: According to your documentation, this should be an ID.
Shouldn't the command, or the dreamcommerceClient check for this?
Or do we want dreamcommerce to define our constraints ( seems a little bit unwanted in certain scenario's )

File: src/LicenseManagerClient.php (hunk 2 above)
Comment (rjd22) [inline src/LicenseManagerClient.php]: That will fail. `(string) false` will also be `(boolean) true` if you do it like you are suggesting.

File: tests/LicenseManagerClientTest.php (hunk 3 above)
Comment (rjd22) [inline tests/LicenseManagerClientTest.php]: There is a single fail flow that will catch them all!!!!!

File: tests/LicenseManagerClientTest.php (hunk 4 above)
Comment (rjd22) [inline tests/LicenseManagerClientTest.php]: Yes this should be a shop ID but an ID is not numeric but a string.

**Comment by lucasvanlierop** [comment]
:+1:

**Comment by arnovr** [comment]
:+1:

**Pull request merged**